backend/app/services/interfaces/user_service.py

```diff
@@ -83,7 +83,7 @@ def get_users(self):
         pass
 
     @abstractmethod
-    def create_user(self, user, auth_id=None, signup_method="PASSWORD"):
+    def create_user(self, user, auth_id=None):
```

Comment:
I'm assuming they added an option for adding SSO, let's actually keep this because imo SSO is easier and safer.